:bookmark: Release 0.10.1 (#145)
maykinmedia/open-api-framework: CHANGELOG.rst

Changelog
=========

0.10.1 (2025-05-23)
-------------------

**Maintenance**

* [#133] Replace ``black``, ``isort`` and ``flake8`` with ``ruff`` and update code-quality workflow

> * Fix ``log_outgoing_requests`` handler when ``django-log-outgoing-requests`` is not active

Reviewer: ```suggestion
* Do not use `save_outgoing_requests` log handler if `LOG_REQUESTS` is set to false
```
